testing-common/src/testFixtures/java/org/hypertrace/agent/testing/TestOpenTelemetryInstaller.java

```diff
@@ -33,13 +34,25 @@ public TestOpenTelemetryInstaller(SpanProcessor spanProcessor) {
   }
 
   @Override
-  public void beforeByteBuddyAgent(Config config) {
+  public void beforeAgent(Config config) {
     OpenTelemetrySdk.builder()
         .setTracerProvider(SdkTracerProvider.builder().addSpanProcessor(spanProcessor).build())
         .setPropagators(ContextPropagators.create(W3CTraceContextPropagator.getInstance()))
         .buildAndRegisterGlobal();
+
+    try {
+      Class<?> agentInitializerClass =
```

Comment:
Could you please document this work-around? Is there upstream issue to handle this properly?

Reply:
No upstream issue here because I'm working on a strategy to get us leveraging more of their test fixtures